components/tools/OmeroWeb/omeroweb/api/api_query.py

```diff
@@ -20,12 +20,36 @@
 """Helper functions for views that handle object trees."""
 
 
-from omero.rtypes import unwrap
+from omero.rtypes import unwrap, wrap
+from omero.sys import ParametersI
 
 from api_marshal import marshal_objects
 from copy import deepcopy
 
 
+def get_child_counts(conn, link_class, parent_ids):
+    """
+    Count child links for the specified parent_ids.
+
+    @param conn:        BlitzGateway
+    @param link_class:  E.g. 'ProjectDatasetLink'
```

Comment:
I know it is code but you should never start a sentence if ``e.g.`` anyway
better to have a description then ``e.g.``